governance/DeprecationMeta.md

```diff
@@ -0,0 +1,27 @@
+# Deprecation Meta
+
+# Overview
+
+This document contains high-level information about O3DE's approach to deprecation.
+
+It details language and terms used when categorizing types of deprecation.
+
+## Philosophy
+
+When modifying existing code that external customers rely on we should strive wherever possible to not break them when making changes.
+
+If code is to be changed or removed, we should let customers know ahead of time with a reasonable notice period (approximately 2 releases (2-3 months))
+
+## Feature Deprecation vs API Deprecation
+
+We define *Feature* deprecation as deprecating (and then replacing or removing) a system or feature (usually user facing).
```

Comment:
To me, feature deprecation should be tied to an RFC. Need plenty of early heads up and probably a notification to TSC/SIG-all

Reply:
Yup this is a good call, I can makes sure this is called out explicitly in the upcoming TBD RFC. This is a good way to broadcast out early things will be going away/changing instead of it being a surprise in a release note